tests/cuda/cuda_point.cu

```diff
@@ -0,0 +1,80 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2016 by the deal.II authors
```

Comment:
Wrong year. Does the script correct the date when we add a new file?

Reply:
The script modifies the year the file was touched last.